umh-core/pkg/communicator/actions/edit-dataflowcomponent.go

```diff
@@ -596,44 +610,50 @@ func (a *EditDataflowComponentAction) waitForComponentToBeActive(ctx context.Con
 							continue
 						}
 
-						if instance.CurrentState == "active" || instance.CurrentState == "idle" {
-							stateMessage := RemainingPrefixSec(remainingSeconds) + fmt.Sprintf("component successfully activated with state '%s', configuration verified", instance.CurrentState)
-							SendActionReply(a.instanceUUID, a.userEmail, a.actionUUID, models.ActionExecuting,
-								stateMessage, a.outboundChannel, models.EditDataFlowComponent)
-							return "", nil
+						// depending on the desired state, we accept different states
+						var acceptedStates []string
+						switch a.state {
+						case models.ComponentStateActive:
+							acceptedStates = []string{"active", "idle"}
```

Comment:
see also above. maybe it makes sense also to combine it in a states.go subfile here in the actions folder as it is the same code as above